package.json

```diff
@@ -49,20 +47,20 @@
     "lint-staged": "^15.2.10"
   },
   "resolutions": {
-    "**/nomnom/underscore": "^1.12.1",
-    "**/core/ember-inline-svg/**/nth-check": "^2.0.1",
-    "**/ember-cli/**/ansi-html": "^0.0.9",
-    "**/ember-cli/testem/fireworm/async": "^2.6.4",
-    "**/@hashicorp/design-system-components/ember-stargate": "^0.6.0",
-    "micromatch": "^4.0.8"
+    "nomnom/underscore": "^1.12.1",
+    "@hashicorp/design-system-components/ember-stargate": "^0.6.0",
+    "micromatch": "^4.0.8",
+    "ember-inline-svg/nth-check": "^2.0.1",
+    "node-gyp": "^10.0.0",
+    "ember-cli/ansi-html": "^0.0.9",
```

Comment:
Looks like `ansi-html` is resolving to `0.0.7` now. I think its because `ember-cli` doesn't have `ansi-html` as a dependency. Looks like its `broccoli-middleware` and `broccoli` (those are deps. of `ember-cli`) that use `ansi-html`.

Reply:
ooh good catch! I think we might not need this resolution anymore, since this didn't resolve correctly and didn't cause any issues

Reply:
I think this [dependabot alert](https://github.com/hashicorp/boundary-ui/security/dependabot/26) will reopen if there's no resolution since versions less than 0.0.8 are vulnerable. We can add two resolutions or just one global resolution
```
"broccoli/ansi-html": "^0.0.9",
"broccoli-middleware/ansi-html": "^0.0.9",
```
or i think this would work too:
```
"ansi-html": "^0.0.9",
```

Reply:
oh thank you for linking that!